subsys/usb/device_next/class/usbd_cdc_ncm.c

```diff
@@ -518,14 +519,15 @@ static int cdc_ncm_acl_out_cb(struct usbd_class_data *const c_data,
 	int ret;
 
 	if (err || buf->len == 0) {
+		LOG_ERR("Bulk OUT transfer error or zero length");
 		net_buf_unref(buf);
 		atomic_clear_bit(&data->state, CDC_NCM_OUT_ENGAGED);
 		return 0;
 	}
 
 	ret = check_frame(data, buf);
 	if (ret < 0) {
-		LOG_DBG("check frame failed (%d)", ret);
+		LOG_ERR("check frame failed (%d)", ret);
```

Comment:
What's the real reason behind this? Is this so unexpected condition? Deliberately malformed packets are really common on networks.

I would drop this commit, otherwise the rest seems fine.

Reply:
It would not be a mailformed network packet, but a mailformed NTB, so this should be logged as error. We do not check Ethernet packets and do not care about them at all.